src/bin/inx-chronicle/migrations/migrate_1.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use async_trait::async_trait;
+use chronicle::db::{mongodb::collections::OutputCollection, MongoDb, MongoDbCollectionExt};
+
+use super::Migration;
+
+pub struct Migrate;
+
+#[async_trait]
+impl Migration for Migrate {
+    const ID: usize = 1;
+    const APP_VERSION: &'static str = "1.0.0-beta.37";
```

Comment:
hmm, why does this have to contain the TO version, and not the FROM app version? That forces us to release that version even if we not really want to due to later decisions.